server/src/test/java/com/objectcomputing/checkins/services/team/member/TeamMemberControllerTest.java

```diff
@@ -453,8 +454,61 @@ void testUpdateTeamMemberThrowExceptionWithInvalidId() {
         String error = Objects.requireNonNull(body).get("message").asText();
         String href = Objects.requireNonNull(body).get("_links").get("self").get("href").asText();
 
-        assertEquals(String.format("Unable to locate teamMember to update with id %s",teamMember.getId()),error);
+        assertEquals(String.format("Unable to locate teamMember to update with id %s", teamMember.getId()), error);
         assertEquals(request.getPath(), href);
         assertEquals(HttpStatus.BAD_REQUEST, responseException.getStatus());
+
+    }
+
+    @Test
+    void testDeleteTeamMemberAsAdmin() {
+        Team team = createDeafultTeam();
+        MemberProfile memberProfile = createADefaultMemberProfile();
+
+        TeamMember teamMember = createDeafultTeamMember(team, memberProfile);
+
+        final HttpRequest<Object> request = HttpRequest.
+                DELETE(String.format("/%s", teamMember.getId())).basicAuth(ADMIN_ROLE, ADMIN_ROLE);
+
+        final HttpResponse<TeamMember> response = client.toBlocking().exchange(request, TeamMember.class);
+
+        assertEquals(HttpStatus.OK, response.getStatus());
     }
+
+    @Test
+    void testDeleteTeamMemberWithoutAdminPrivilege() {
+        Team team = createDeafultTeam();
+        MemberProfile memberProfile = createADefaultMemberProfile();
+
+        TeamMember teamMember = createDeafultTeamMember(team, memberProfile);
+
+        final HttpRequest<Object> request = HttpRequest.
+                DELETE(String.format("/%s", teamMember.getId())).basicAuth(MEMBER_ROLE, MEMBER_ROLE);
+
+        HttpClientResponseException responseException = assertThrows(HttpClientResponseException.class,
+                () -> client.toBlocking().exchange(request, Map.class));
+
+        assertNotNull(responseException.getResponse());
+        assertEquals(HttpStatus.UNAUTHORIZED, responseException.getStatus());
+
+    }
+
+    @Test
+    void testDeleteInvalidTeamMemberAsAdmin() {
+        Team team = createDeafultTeam();
+        MemberProfile memberProfile = createADefaultMemberProfile();
+
+        TeamMember teamMember = createDeafultTeamMember(team, memberProfile);
+        teamMember.setId(UUID.randomUUID());
+
+        final HttpRequest<Object> request = HttpRequest.
+                DELETE(String.format("/%s", teamMember.getId())).basicAuth(ADMIN_ROLE, ADMIN_ROLE);
+
+        HttpClientResponseException responseException = assertThrows(HttpClientResponseException.class,
+                () -> client.toBlocking().exchange(request, Map.class));
+
+        assertNotNull(responseException.getResponse());
+        assertEquals(HttpStatus.NOT_FOUND, responseException.getStatus());
+    }
+
```

Comment:
Can we add a test case to make sure that team lead can remove a member?

Reply:
done